docs/must-gather.md

```diff
@@ -0,0 +1,33 @@
+# Must-Gather
+
+## Collect logs
+
+We are collecting the following set of data with best-effort
+
+- lca
```

Comment:
Do we collect logs and other relevant data from the unbooted stateroot?

Reply:
Nothing special really...so just the dump of  `rpm-ostree status --json` which shows some good data about the stateroots. Are you thinking of anything specific that i can add now?

Reply:
I'm referring to /var/log content from the unbooted stateroot. I expect that being able to debug issues would mean we'd at least need the log files from the unbooted stateroot, which wouldn't be collected by anything else. Possibly its /etc content as well, and maybe other bits.

Unless that content should be collected by something else, @browsell . Certainly there'd be nothing collecting it today. But if we're asked to debug an upgrade failure, or some other issue, this would be needed.

Reply:
Now collecting everything under unbooted stateroot's `/var/log` + everything under`/etc`